opentasks/src/main/res/raw/sharetask.template

```diff
@@ -0,0 +1,18 @@
+{{ $task.title }}
+============
+{% if $task.description %}
+{{ $task.description }}
+{% end %}{% if $task.checklist %}{% for item in $task.checklist %}
+[{% if item.checked %}X{% else %} {% end %}] {{ item.text }}{% end %}
```

Comment:
Ẁith the latest version of carrot you can replace
```
[{% if item.checked %}X{% else %} {% end %}]
```
with
```
{{ item.checked and "X" or " " }}
```
which is easier to read and does the same otherwise.
This works because boolean operators now return the original value which decided on the outcome of the operation.

Reply:
Wow, quite advanced evaluation syntax supported now. I've update it.